### PR TITLE
MMA-3573. Update from upstream

### PR DESCRIPTION
**Pull request by Silviu-Surcica**
We need to release tag v1.4.0 which is compatible with debian buster.

### CODE REVIEW

**Review by tonyandrewmeyer — APPROVED** [review]
I assume this is just a fetch and merge from upstream? Nothing I can really review in that case, QA needs to handle making sure all works fine still.

**Review by bitsikas — APPROVED** [review]

File: metadata.cfg [hunk 1]
```diff
@@ -44,26 +19,22 @@ classifiers:
         Operating System :: Unix
         Programming Language :: C
         Programming Language :: Python
-        Programming Language :: Python :: 2
-        Programming Language :: Python :: 2.7
```
Comment (tonyandrewmeyer) [inline metadata.cfg:48]: This seems like an issue? Unless we are not planning to update until the Py3 project is done.

Comment (bitsikas) [reply metadata.cfg:48]: I think that for our own needs we'll publish the last compatible tags in version branches

Comment (Silviu-Surcica) [reply metadata.cfg:48]: As I talked with kostas we can release from the tag v.1.4.0 which solves our compatibility issues with debian buster.

**Pull request merged**